Library/DataSource/ValueCellDataSource.swift

```diff
@@ -71,21 +71,45 @@ open class ValueCellDataSource: NSObject, UICollectionViewDataSource, UITableVie
   }
 
   /**
+   Inserts a single value at the beginning of the section specified.
+
+   - parameter value:     A value to prepend.
+   - parameter cellClass: The type of cell associated with the value.
+   - parameter section:   The section to append the value to.
+
+   - returns: The index path of the prepended row.
+   */
+  @discardableResult
+  public final func prependRow <
```

Comment:
If we're happy to return the discardable `IndexPath` here, should we add it to the other functions for consistency?